tools/perf_checker/perf_checker.sh

```diff
@@ -0,0 +1,41 @@
+.sh
+#!/bin/sh
+set -euox pipefail
+
+# Installing the requirements
+python -m venv perf
+source perf/bin/activate
+pip install psutil
+pip install gitpython
+pip install somacore
+pip install tiledbsoma
+pip install cellxgene_census
+
+# Installing mount-s3
+sudo wget https://s3.amazonaws.com/mountpoint-s3-release/latest/x86_64/mount-s3.deb
+sudo apt install -y ./mount-s3.deb
+
+# Setting up mount-s3
+mkdir ./mount-s3
```

Comment:
would rename to indicate its contents instead? could be same as bucket name: `census-profiler-tests`. Would also add comment explaining that this is necessary to persist the profiling run data that are performed below.

Reply:
Great ideas both